<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>lome6-gui</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/If.ico</normaloff>:/new/prefix1/If.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Host:</string>
    </property>
    <property name="buddy">
     <cstring>lineHost</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>5</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Port:</string>
    </property>
    <property name="buddy">
     <cstring>linePort</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>69</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Username:</string>
    </property>
    <property name="buddy">
     <cstring>lineUsername</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>P&amp;assword:</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
    <property name="buddy">
     <cstring>linePassword</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineHost">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>00000; </string>
    </property>
    <property name="text">
     <string>2701</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxAuth">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>223</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Use authentification</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUsername">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>65</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePassword">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>65</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Disconnect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>221</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabSystem">
     <attribute name="title">
      <string>System Control</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>System power state:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSystemPowerState">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>off</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>lome6 Uptime:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUptimeLome6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Uptime:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUptime">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPower">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Press Power &amp;Button</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPowerLong">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Press Power Button (&amp;Long)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonReset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Press &amp;Reset Button</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabThermal">
     <attribute name="title">
      <string>Thermal Monitoring</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RAM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Supply:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Southbridge:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Air Intake:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTempCPU">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTempRAM">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTempPSU">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTempSB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTempAIR">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabConfig">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVersion">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>160</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>MAC Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IP Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Netmask:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>115</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Gateway:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>145</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NTP Server:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRestart">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Restart lome6</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMAC">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>HH:HH:HH:HH:HH:HH; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineIP">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNM">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineGW">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNTP">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000; </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
